src/test/java/pl/stormit/eduquiz/statistic/quizstatistic/QuizStatisticFacadeImpIntegrationTest.java

```diff
@@ -19,17 +20,18 @@
 
 import java.time.LocalDateTime;
 import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Also not a big deal, but earlier we rather used single imports instead of " .* " that import everything